src/dhtproto/client/request/internal/RemoveChannel.d

```diff
@@ -0,0 +1,324 @@
+/*******************************************************************************
+
+    Client DHT RemoveChannel v0 request handler.
+
+    Copyright:
+        Copyright (c) 2017 sociomantic labs GmbH. All rights reserved.
```

Comment:
Copyright 2018.

Reply:
(in other files as well)

Reply:
Well, this code was written in 2017 ;)

Reply:
Yeah, I was wondering about that. All right, as long as I have a precedent to refer to in the reviews  I receive ;-)